contracts/governance/extensions/GovernorTimelockCompound.sol

```diff
@@ -119,7 +121,9 @@ abstract contract GovernorTimelockCompound is IGovernorTimelock, Governor {
         bytes32 /*descriptionHash*/
     ) internal virtual override {
         uint256 eta = proposalEta(proposalId);
-        require(eta > 0, "GovernorTimelockCompound: proposal not yet queued");
+        if (eta == 0) {
+            revert GovernorMissingETA(proposalId);
```

Comment:
I'd rather have a `GovernorProposalNotQueued(proposalId)` error. Missing ETA is not really explicit here

Reply:
Implemented, thanks for the suggestion!